contribute.md

```diff
@@ -70,9 +70,9 @@ If you need to reproduce the site locally and/or make extensive changes, you can
 
 ### Forking this Repository
 
-First check to see if you already have a fork of this repository:
+First check to see if you already have a fork of this repository at:
 
-[https://github.com/<your username>/design][]
+https://github.com/{your github username}/design
```

Comment:
I think it's fine to leave it outside of the `[]()` syntax (which I'm assuming was intended here, but resulted in a typo).